hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -56,10 +60,10 @@
  */
 public class Token {
```

Comment:
Lets place the operations one after another:

1. 2x mints
2. 2x burns
3. 2x wipes